src/Interpreters/Context.cpp

```diff
@@ -2461,7 +2461,8 @@ AsyncLoader & Context::getAsyncLoader() const
                 }
             },
             /* log_failures = */ true,
-            /* log_progress = */ true);
+            /* log_progress = */ true,
+            /* log_events = */ true);
```

Comment:
why add a flag at all if it is always true and non-changeable? (as I see it is set to false only for tests, so in general does not seem to make sense to add a flag unless this could be changeable by a setting).